cmake/helpers/CheckDependentLibraries.cmake

```diff
@@ -576,7 +576,7 @@ gdal_check_package(GEOS "Geometry Engine - Open Source (GDAL core dependency)" R
 )
 gdal_check_package(HDF4 "Enable HDF4 driver" CAN_DISABLE)
 
-define_find_package2(KEA libkea/KEACommon.h kea)
+define_find_package2(KEA libkea/KEACommon.h kea;libkea)
```

Comment:
why/when is that needed ? according to https://cmake.org/cmake/help/latest/command/find_library.html "Each library name given to the NAMES option is first considered as a library file name and then considered with platform-specific prefixes (e.g. lib) and suffixes (e.g. .so)."

Reply:
On Windows AFAICT `lib` isn't one of the prefixes that is tried but the library is called `libkea.lib` on Windows.

Reply:
@rouault The file name is: 
`libkea.lib`.
But the tested patterns for Windows are:
`kea` (the "name")
`kea.lib` (name with prefix and suffix).
(This is a problem when Windows libs try to simulate POSIX name style.